planetiler-core/src/main/java/com/onthegomap/planetiler/mbtiles/Mbtiles.java

```diff
@@ -929,6 +930,16 @@ public TileArchiveMetadata get() {
       String[] center = map.containsKey(TileArchiveMetadata.CENTER_KEY) ?
         map.remove(TileArchiveMetadata.CENTER_KEY).split(",") : null;
       var metadataJson = MetadataJson.fromJson(map.remove("json"));
+
```

Comment:
I since the spec requires gzip compression, I wonder if we should log a warning in the set method if tile compression != gzip?